### PR TITLE
Unsuspend publish-over-ssh since 1.24

### PR DESCRIPTION
**Pull request by Wadeck**
FYI @gmcdonald @olamy 

Corrections:
- SECURITY-2287, https://github.com/jenkinsci/publish-over-ssh-plugin/pull/50 and https://github.com/jenkinsci/publish-over-ssh-plugin/pull/59
- SECURITY-2290, https://github.com/jenkinsci/publish-over-ssh-plugin/pull/51 and https://github.com/jenkinsci/publish-over-ssh-plugin/pull/58
- SECURITY-2291, https://github.com/jenkinsci/publish-over-ssh-plugin/pull/52
- SECURITY-2307, https://github.com/jenkinsci/publish-over-ssh-plugin/pull/54

Fix version with the fixes: 1.24

With its companion to remove the entry in the https://www.jenkins.io/security/plugins/#suspensions list: https://github.com/jenkins-infra/jenkins.io/pull/4940

### CODE REVIEW

File: resources/artifact-ignores.properties [hunk 1]
```diff
@@ -677,7 +677,45 @@ carbonetes-serverless-container-scanning-and-policy-compliance-1.8.1
...
+publish-over-ssh-1.17
+publish-over-ssh-1.18
+publish-over-ssh-1.19
+publish-over-ssh-1.19.1
+publish-over-ssh-1.20
+publish-over-ssh-1.20.1
+publish-over-ssh-1.21
+publish-over-ssh-1.22
+publish-over-ssh-1.23
+# fixes were proposed since publish-over-ssh-1.24
```
Comment (daniel-beck) [inline resources/artifact-ignores.properties:718]:
```suggestion
```

No need to spam the suspension list here.

**Review by daniel-beck — CHANGES_REQUESTED** [review]
This also needs to update warnings regexes.

**Review by timja — APPROVED** [review]

File: resources/artifact-ignores.properties [hunk 2]
```diff
@@ -677,7 +677,7 @@ carbonetes-serverless-container-scanning-and-policy-compliance-1.8.1
 # https://www.jenkins.io/security/advisory/2022-01-12/
 batch-task = https://www.jenkins.io/security/plugins/#suspensions
 debian-package-builder = https://www.jenkins.io/security/plugins/#suspensions
-publish-over-ssh = https://www.jenkins.io/security/plugins/#suspensions
+# publish-over-ssh => fixes were proposed since publish-over-ssh-1.24, adjust warnings
```
Comment (daniel-beck) [inline resources/artifact-ignores.properties:680]:
```suggestion
```

This is what commit history is for.

**Comment by Wadeck** [comment]
Adjusted using the feedbacks received, merging.

**Pull request merged**